AsyncDisplayKit.podspec

```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |spec|
       'Base/*.{h,m}'
   ]
 
-  spec.frameworks = 'AssetsLibrary'
+  spec.frameworks = 'AssetsLibrary','MapKit'
   spec.weak_frameworks = 'Photos'
```

Comment:
Looks like we might not hear back in time for 1.9.2, which I hope to ship at the end of tomorrow.  We don't have to include this in that release, although I'd like to, as we'll start using it in Pinterest :).

If you have time to research the Cocoapods feature itself, please do.  I bet this is possible, but I'm wondering how this works for a given user.  Maybe you could try adding it to weak frameworks locally, and add a map node to one of the example projects that doesn't link MapKit, and see what happens.  Presumably you'll get a linker error, and I think that would be totally fine to expect users to have to manually have linked MapKit if they want to use this feature.